trainsim-client/nginx.conf

```diff
@@ -26,9 +26,12 @@ http {
         gzip_types text/css application/javascript application/json image/svg+xml;
         gzip_comp_level 9;
         etag on;
-        location /api {
+        location /api/stops {
```

Comment:
I think we should use /api/\<microservice name\>. So we can have this as `/api/itinerary`

Reply:
resolved.